mirascope/anthropic/calls.py

```diff
@@ -87,6 +87,7 @@ def call(
         )
         return AnthropicCallResponse(
             response=message,
+            user_message_param=messages[-1] if messages[-1]["role"] == "user" else None,
```

Comment:
refactor to utility fn that all providers share

Reply:
Done.